README.rst

```diff
@@ -1,9 +1,22 @@
-.. image:: http://pybee.org/project/projects/bridges/batavia/batavia-72.png
-    :target: http://pybee.org/batavia
+.. image:: http://pybee.org/project/projects/bridges/batavia/batavia.png
```

Comment:
What procedure did you use to merge up this patch? It's a little confusing because this PR contains a bunch of code that has been reviewed and included elsewhere; `git fetch upstream && git merge upstream/master` should have introduced a merge commit, but not all the individual commits.